### DISCUSSION TITLE
IdeaStatica - Parameric Templates help

### DISCUSSION THREAD

**Discussion by MapliteStudio** [idea-statica/ideastatica-public#335] [Help Q&A] [answered]
Dear All, Recently I started exploring Idea Statica Parametric and need to understand few basics like how to use these templates mean workflow of these templates. Could somebody help!!

![Idea Statica Parametric](https://github.com/idea-statica/ideastatica-public/assets/146530224/3d53cfae-e083-4355-93a5-a5b3f37d4362)

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @MapliteStudio, You use the _Model Properties_ tab to link the Parameters you have set up with available inputs of the Connection App. Can you please click on the _Model Properties_ tab and show what you have there? 

Just by looking at the model in the image, it does not look like you have any operations defined. You need to have basic operations that you can use parameters to drive. 

You can link parameters to operation properties (model properties) by clicking the link button on the parameters tab or by managing them in the Model Properties.

 
![image](https://github.com/idea-statica/ideastatica-public/assets/93578548/e1830ce8-9705-4656-92d3-06f65447e9ad)
